Uplift/Views/GymDetail/Facilities/FacilitiesHoursCell.swift

```diff
@@ -12,7 +12,7 @@ class FacilitiesHoursCell: UICollectionViewCell {
 
     // MARK: - Public data vars
     static var baseHeight: CGFloat {
-        return Constraints.headerHeight + Constraints.timesPadding + Constraints.bottomPadding
+        return Constraints.headerHeight
```

Comment:
make sure we remove the `timesPadding` and `bottomPadding` from `Constraints` if they're not used anywhere else